app/controllers/catalog_controller.rb

```diff
@@ -74,11 +74,11 @@ def self.modified_field
     config.add_index_field solr_name('date_created', :stored_searchable), itemprop: 'dateCreated'
     config.add_index_field solr_name('rights_statement_label', :stored_searchable), label: 'Rights Statement', link_to_search: solr_name('rights_statement_label', :facetable)
     config.add_index_field solr_name('license_label', :stored_searchable), label: 'License', link_to_search: solr_name('license_label', :facetable)
-    config.add_index_field solr_name('type_label', :stored_searchable), label: 'Resource Type', link_to_search: solr_name('type_label', :facetable)
     config.add_index_field solr_name('file_format', :stored_searchable), link_to_search: solr_name('file_format', :facetable)
     config.add_index_field solr_name('identifier', :stored_searchable), helper_method: :index_field_link, field_name: 'identifier'
     config.add_index_field solr_name('embargo_release_date', :stored_sortable, type: :date), label: 'Embargo release date', helper_method: :human_readable_date
     config.add_index_field solr_name('lease_expiration_date', :stored_sortable, type: :date), label: 'Lease expiration date', helper_method: :human_readable_date
+    config.add_index_field solr_name('type_label', :stored_searchable), label: 'Resource Type', link_to_search: solr_name('type_label', :facetable), if: false
```

Comment:
`if: false` what would be evaluating to false in this context?

Reply:
Choosing whether to display the field in the index/search page, similar to title above. We add it as an index field here to easily render it later (with the icon at the bottom).